lib/pure/coro.nim

```diff
@@ -32,6 +32,10 @@ import lists
 include system/timers
 
 const defaultStackSize = 512 * 1024
+const useOrcArc = defined(gcOrc) or defined(gcArc)
```

Comment:
``gcArc`` is defined for ``--gc:orc`` too.

Reply:
`nimGC_setStackBottom` is defined in `mm.nim` already, so I can't define them in `arc.nim`. I have to define them in `coro.nim`. So I need `gcOrc` .